From: Andrea Zagli <azagli@libero.it>
Date: Sat, 28 May 2016 10:58:15 +0200
Subject: Backport function g_key_file_save_to_file.

---
 src/session.c | 22 ++++++++++++++++++++++
 1 file changed, 22 insertions(+)

diff --git a/src/session.c b/src/session.c
index 2c064f0..89403c3 100644
--- a/src/session.c
+++ b/src/session.c
@@ -112,6 +112,28 @@ zak_cgi_session_init (ZakCgiSession *zak_cgi_session)
 	priv->minutes = MINUTES_DEFAULT;
 }
 
+gboolean
+g_key_file_save_to_file (GKeyFile     *key_file,
+                         const gchar  *filename,
+                         GError      **error)
+{
+  gchar *contents;
+  gboolean success;
+  gsize length;
+
+  g_return_val_if_fail (key_file != NULL, FALSE);
+  g_return_val_if_fail (filename != NULL, FALSE);
+  g_return_val_if_fail (error == NULL || *error == NULL, FALSE);
+
+  contents = g_key_file_to_data (key_file, &length, NULL);
+  g_assert (contents != NULL);
+
+  success = g_file_set_contents (filename, contents, length, error);
+  g_free (contents);
+
+  return success;
+}
+
 /**
  * zak_cgi_session_new:
  * @zakcgimain:
